contrib/apelscripts/condor_batch.sh

```diff
@@ -32,6 +23,5 @@ TZ=GMT condor_history -constraint "$CONSTR" \
     -format "%d|" ResidentSetSize_RAW \
     -format "%d|" ImageSize_RAW \
     -format "%d|" RequestCpus \
-    -format "%v|" ${SCALING_ATTR} \
-    -format "\n" EMPTY  | sed -e "s/undefined|$/1.0|/" > $OUTPUT_FILE
+    "${HISTORY_SUFFIX}" > "$(condor_ce_config_val APEL_OUTPUT_DIR)/$output"
```

Comment:
This doesn't look right to me.  Why is `"${HISTORY_SUFFIX}"` quoted?

Hint: I think maybe you want to construct it as an array instead of a single string.